lib/src/fluid_nav_bar_style.dart

```diff
@@ -19,7 +19,7 @@ import 'package:flutter/widgets.dart';
 /// ```
 /// {@end-tool}
 @immutable
-class FluidNavBarStyle extends Diagnosticable {
+class FluidNavBarStyle with Diagnosticable {
```

Comment:
As you try to mixin `Diagnosticable`, it's probably better to use directly the mixin, no? 
```suggestion
class FluidNavBarStyle with DiagnosticableMixin {
```

Reply:
Maybe haha.I am not very familiar with this.